<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="73097" started-at="2022-12-29T13:59:40 IST" finished-at="2022-12-29T14:00:53 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="73097" started-at="2022-12-29T13:59:40 IST" finished-at="2022-12-29T14:00:53 IST">
      <class name="selenium.TestCase">
        <test-method status="PASS" signature="initialization()[pri:0, instance:selenium.TestCase@3d5c822d]" name="initialization" is-config="true" duration-ms="1651" started-at="2022-12-29T14:00:39 IST" finished-at="2022-12-29T14:00:41 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method status="PASS" signature="f(java.lang.String,java.lang.String)[pri:0, instance:selenium.TestCase@3d5c822d]" name="f" duration-ms="7762" started-at="2022-12-29T14:00:41 IST" data-provider="getdata" finished-at="2022-12-29T14:00:49 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[thriveni@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method status="PASS" signature="f(java.lang.String,java.lang.String)[pri:0, instance:selenium.TestCase@3d5c822d]" name="f" duration-ms="4011" started-at="2022-12-29T14:00:49 IST" data-provider="getdata" finished-at="2022-12-29T14:00:53 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[thriveni@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[thriv4526]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method status="PASS" signature="close()[pri:0, instance:selenium.TestCase@3d5c822d]" name="close" is-config="true" duration-ms="175" started-at="2022-12-29T14:00:53 IST" finished-at="2022-12-29T14:00:53 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- selenium.TestCase -->
      <class name="selenium.AssertionValidation">
        <test-method status="PASS" signature="initialization()[pri:0, instance:selenium.AssertionValidation@429bffaa]" name="initialization" is-config="true" duration-ms="58971" started-at="2022-12-29T13:59:40 IST" finished-at="2022-12-29T14:00:39 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method status="FAIL" signature="getHeader()[pri:0, instance:selenium.AssertionValidation@429bffaa]" name="getHeader" duration-ms="1" started-at="2022-12-29T14:00:53 IST" finished-at="2022-12-29T14:00:53 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at selenium.AssertionValidation.getHeader(AssertionValidation.java:47)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getHeader -->
        <test-method status="FAIL" signature="getTitle()[pri:0, instance:selenium.AssertionValidation@429bffaa]" name="getTitle" duration-ms="83" started-at="2022-12-29T14:00:53 IST" finished-at="2022-12-29T14:00:53 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [FEATURE COURSES] but found [FEATURED COURSES]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [FEATURE COURSES] but found [FEATURED COURSES]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at selenium.AssertionValidation.getTitle(AssertionValidation.java:39)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getTitle -->
        <test-method status="PASS" signature="close()[pri:0, instance:selenium.AssertionValidation@429bffaa]" name="close" is-config="true" duration-ms="307" started-at="2022-12-29T14:00:53 IST" finished-at="2022-12-29T14:00:53 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- selenium.AssertionValidation -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
